app/projectsproxymodel.h

```diff
@@ -52,6 +57,7 @@ class ProjectsProxyModel : public QSortFilterProxyModel
     ProjectsModel *mModel = nullptr; // not owned by this, needs to be set in order to proxy model to work
     ProjectsModel::ProjectModelTypes mModelType = ProjectsModel::EmptyProjectsModel;
     QString mSearchExpression;
+    bool mactiveProjectAlwaysFirst = false;
```

Comment:
just a code convention
```suggestion
    bool mActiveProjectAlwaysFirst = false;
```